tests/Aspire.Playground.Tests/Aspire.Playground.Tests.csproj

```diff
@@ -0,0 +1,52 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>$(NetCurrent)</TargetFramework>
+
+    <IsTestProject>true</IsTestProject>
+    <!-- no docker support on helix/windows yet -->
+    <RunTestsOnHelix Condition="'$(OS)' != 'Windows_NT'">true</RunTestsOnHelix>
+    <SkipTests Condition="'$(OS)' == 'Windows_NT'">true</SkipTests>
+
+    <DeployOutsideOfRepoSupportFilesRelativeDir>staging-archive\</DeployOutsideOfRepoSupportFilesRelativeDir>
+
+    <TestArchiveTestsDir>$(TestArchiveTestsDirForBuildOnHelixTests)</TestArchiveTestsDir>
+
+    <PlaygroundSourceDir>$(MSBuildThisFileDirectory)..\..\playground\</PlaygroundSourceDir>
+    <TestsSharedDir>$(MSBuildThisFileDirectory)..\Shared\</TestsSharedDir>
+    <RunSettingsFilePath>$(MSBuildThisFileDirectory).runsettings</RunSettingsFilePath>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="$(TestsSharedDir)WorkloadTesting\TestOutputWrapper.cs" />
+    <Compile Include="$(TestsSharedDir)WorkloadTesting\EnvironmentVariables.cs" />
+    <Compile Include="$(TestsSharedDir)Logging\*.cs" />
+
+    <AspireProjectOrPackageReference Include="Aspire.Hosting" />
```

Comment:
Do we rebuild the Aspire.Playground.Tests in Helix? If so, why? Why don't we build this project on the build machine, and then run the already built tests on Helix?

Reply:
hmm I think this can be reworked so this project builds on the build machine. Then we can build the playground projects on helix, and point the tests to the bindir for these.

Reply:
This will need some reworking. I have added it as a follow up task in https://github.com/dotnet/aspire/issues/5210 . We can get the tests checked-in before that.